{"name": "test_wrong_pwd", "status": "passed", "description": "密码输入6次错误", "attachments": [{"name": "log", "source": "2a095422-9be4-4ae6-9a98-ce41e0184c6e-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "e6f1dc15-4008-4d61-a2c9-35088d6b0900-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "9720aee9-c632-4e43-843f-a7af4b4bb056-attachment.txt", "type": "text/plain"}], "start": 1631518042600, "stop": 1631518043891, "uuid": "77431ced-1e8a-421e-96a1-badb98d7374a", "historyId": "7b9480ebb1961c7c14cb69cc7e65ba50", "testCaseId": "e4c5762512a76c681290f066bcc0e861", "fullName": "test_case.Login_case.TestLogin#test_wrong_pwd", "labels": [{"name": "feature", "value": "登录模块"}, {"name": "story", "value": "参数填写不正确，登录失败"}, {"name": "severity", "value": "critical"}, {"name": "parentSuite", "value": "test_case"}, {"name": "suite", "value": "Login_case"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-KSOAST3"}, {"name": "thread", "value": "34540-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.Login_case"}]}